ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-main.yaml

```diff
@@ -66,6 +66,10 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-serial
+- as: e2e-aws-capi
```

Comment:
Tech preview jobs usually include a -techpreview suffix, so TRT knows these are experimental

Reply:
should I change the workflow name too or step name is enough?

Reply:
Whatever you prefer, the name of the job is all that matters to TRT

Reply:
thanks, I fixed the name

Reply:
Thank you!